packages/package/src/typescript.js

```diff
@@ -28,7 +28,13 @@ export async function emit_dts(input, output, final_output, cwd, alias, files, t
 	const require = createRequire(import.meta.url);
 	const pkg = load_pkg_json(cwd);
 	const svelte_dep = pkg.peerDependencies?.svelte || pkg.dependencies?.svelte || '3.0';
-	const no_svelte_3 = !semver.intersects(svelte_dep, '^3.0.0');
+	let no_svelte_3 = true;
+	try {
+		no_svelte_3 = !semver.intersects(svelte_dep, '^3.0.0');
+	} catch (e) {
+		if (!(e instanceof Error)) throw e;
+		console.info(`Can't check svelte version ${svelte_dep}`, e.message);
```

Comment:
this should probably be a warning?
```suggestion
		console.warn(`Can't check svelte version ${svelte_dep}`, e.message);
```